src/guide/sponsoring-a-change.md

```diff
@@ -0,0 +1,43 @@
+# Sponsoring a Change
+
+New developers in the OpenJDK community doesn't have the permissions needed to push changes to the repositories. This is a feature that ensures that all developers get familiar with the code, processes, and community before being allowed to actually make changes. To get the first changes in, the new Contributor needs the help of a Sponsor. The Sponsor's role is to offer constructive advice and eventually push the sponsored contribution into the repository.
```

Comment:
s/doesn't/don't/ in first sentence.

This definition of sponsor seems to be from the old days when you basically needed someone from Sun/Oracle to help shepherd your change through. These days any of the reviewers will offer advice and may become the eventual sponsor of the commit. So I don't think this section really applies any more i.e. sponsoring-a-change.md doesn't need to exist.

Reply:
I fixed the typo.

It is indeed a text from the old days. Sponsoring is still a thing though so we should have some description of what it means and what to expect from a sponsor. It's a fairly short section, and it may be that some of it can be cut or rephrased to better suit the present. If it becomes much shorter it may fit better as a sub-section under "Contributing to an OpenJDK Project".
Shortening this text is not at the top of my priority list right now though. I just want to get something that is good enough to change the "Sponsoring" link in the OpenJDK sidebar to refer to the guide. That said, if someone else takes a stab at making this text even more up to date I'd be happy to review and sponsor that change :-)

Reply:
The point is that "sponsor" is merely  some committer that now types /sponsor in your PR. The days when you needed to find a sponsor upfront and they needed to shepherd your change through the system (and run testing etc) are long gone.

Reply:
Is that really the case? You would still need someone to run proper testing on multiple platforms etc, so even though the actual integration is easier today the work up to the integration point is basically the same, isn't it?

Reply:
If you can't test a change adequately yourself and need some help, that is true whether you are an Author or Committer, or even Reviewer. Test help is not an integral part of sponsorship.